config/sync/user.role.administrator.yml

```diff
@@ -86,7 +85,6 @@ permissions:
   - 'administer nodes'
   - 'administer openid connect clients'
   - 'administer orphaned events entities'
-  - 'administer paragraphs categories'
```

Comment:
this just confused the editors, as they cant edit the paragraphs to assign categories.